traitsui/qt4/extra/led_editor.py

```diff
@@ -0,0 +1,20 @@
+from pyface.qt import QtGui
+from traitsui.qt4.editor import Editor
+from traitsui.basic_editor_factory import BasicEditorFactory
+
+
+class _LEDEditor(Editor):
+
+    def init(self, parent):
+        self.control = QtGui.QLCDNumber()
+        self.control.setSegmentStyle(QtGui.QLCDNumber.Flat)
+        self.set_tooltip()
+
+    def update_editor(self):
+        self.control.display(self.str_value)
+
+
+class LEDEditor(BasicEditorFactory):
+
+    # The editor class to be created:
```

Comment:
```suggestion
    #: The editor class to be created:
```
The `#:` means that the comment gets picked up by Sphinx auto-documenters.

Reply:
Not sure if this has to be part of the documentation, but changed.